android/libraries/rib-base/src/main/java/com/badoo/ribs/util/RIBs.kt

```diff
@@ -0,0 +1,113 @@
+package com.badoo.ribs.util
+
+import android.support.annotation.VisibleForTesting
+import android.util.Log
+import com.badoo.ribs.android.requestcode.RequestCodeBasedEventStream
+
+object RIBs {
```

Comment:
I think I also kotlinified this class in my last PR, and added some functionality. Not sure why it doesn't complain, so please recheck.

Reply:
Your RIBs class was in a different package. Fixed